src/controllers/customercontroller.js

```diff
@@ -0,0 +1,70 @@
+class CustomerController extends Controller
+{
+    view()
+    {
+        this._customerHelper();
+    }
+
+    generateOrder()
+    {
+        var products = this.constructor._makeOrder();
```

Comment:
I see what you mean! I'll move that around, I think makeOrder should actually be in the base controller, as there are more using it.

I'll move it around in the coming days!